java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -201,4 +203,142 @@ public CompletableFuture<String> set(String key, String value, SetOptions option
                         .build();
         return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
     }
+
+    /**
+     * Decrements the number stored at `key` by one. If `key` does not exist, it is set to 0 before
+     * performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/decr/">redis.io</a> for details.
+     * @param key - The key to decrement its value.
+     * @return the value of `key` after the decrement. An error is raised if `key` contains a value of
```

Comment:
```suggestion
     * @return The value of `key` after the decrement. An error is raised if `key` contains a value of
```

Keep the casing consistend across the docs (update below too)